ocs_ci/deployment/fusion_access.py

```
@@ -0,0 +1,300 @@
+    def create_namespace_and_operator_group(self):
+        """
+        Create the ibm-fusion-access Namespace and its OperatorGroup.
+
+        If the Namespace already exists the step is skipped (the OperatorGroup
+        is assumed to already be present as well since both objects live in the
+        same YAML manifest).
+        """
+        ns_ocp = OCP(kind="Namespace")
+        if ns_ocp.is_exist(resource_name=self.namespace):
+            logger.info(
+                f"Namespace '{self.namespace}' already exists, skipping creation"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)